sdcm/nemesis.py

```diff
@@ -437,7 +441,9 @@ def approach_enospc():
                 self.log.error(str(details))
             return search_database_enospc(node) > orig_errors
 
-        with DbEventsFilter(type='NO_SPACE_ERROR'), DbEventsFilter(type='BACKTRACE', line='No space left on device'):
+        with DbEventsFilter(type='NO_SPACE_ERROR'), \
+                DbEventsFilter(type='BACKTRACE', line='No space left on device'), \
```

Comment:
do we really have a backtrace on No space left on device?

Reply:
yes, on some error resaulting fron no disk space, there's a backtrace
here's an example from last longevity run:
```
2019-04-27 07:53:12,572 remote           L0203 DEBUG| [10.0.213.150] [stdout] Apr 27 07:53:12 warning|  [shard 10] seastar - Exceptional future ignored: storage_io_error (Storage I/O error: 28: No space left on device), backtrace:   0x7efc58
```